src/components/Navbar.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+
+import { Link } from "react-router-dom";
+
+const Navbar = () => {
+    return (
+        <nav className="bg-trip-brown-200 p-6 -xl w-full shadow-lg font-trip-main"> 
+            <div className="flex justify-between items-center">
+               
+           <div> 
+            <a href = "/" className= "text-amber--200 text-2xl hover:text-gray-300">TripWise</a>
```

Comment:
For the `a` tags, use the Link component from Next.js. Refer to the link I sent in the previous comment. Also it seems that you have a typo on `text-amber--200`. 
```suggestion
            <a href = "/" className= "text-amber-200 text-2xl hover:text-gray-300">TripWise</a>
```